app/models/office.rb

```diff
@@ -0,0 +1,9 @@
+class Office < ApplicationRecord
+  validates :title, presence: true
+  validates :description, presence: true
+  validates :area, presence: true
+  validates :occupancy, presence: true, numericality: { only_integer: true }
+  validates :images, presence: true
+  validates :basic_price, presence: true, numericality: true
+  validates :address, presence: true
+end
```

Comment:
[Optional] It's a good idea make all fields as required but sometimes it makes negative UX, i.e., area my considered as an alternative for the occupancy.

Reply:
@jaferIdrees , can you elaborate on this well.

Reply:
too much validations you mean right and that gives bad user experience?